src/components/FooterComponent/Footer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import style from './footer.scss';
+
+const Footer = () => (
+  <footer>
+    <div className={style.container}>
+      <div className={style.information}>
+        <div>
+          <h4>Follow Us</h4>
+          <p>
+            <img src="https://res.cloudinary.com/allebd/image/upload/v1567001062/dahlia/icons8-facebook-f-50.png" alt="facebook logo" />
+            <img src="https://res.cloudinary.com/allebd/image/upload/v1567001069/dahlia/icons8-google-50.png" alt="google logo" />
+          </p>
+        </div>
+        <div>
+          <h4>About Us</h4>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud
+          </p>
+        </div>
+        <div>
+          <h4>Contact Us</h4>
+          <p>
```

Comment:
Change this to a `div`. I am getting a react error in the console.

Reply:
Noted, would update it